arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -605,6 +605,75 @@
         }
       }
     },
+    "UpgradeDetails": {
+      "description": "Details for upgrading vault.",
+      "type": "object",
+      "properties": {
+        "operationId": {
+          "description": "ID of the vault upgrade operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "startTimeUtc": {

```

Comment:
Just so I understand, do you mean that the service itself works with a plain string instead of a formatted date-time one?